RandomError1.py

```diff
@@ -0,0 +1,226 @@
+        dankhold=1
+        totalbase=0
+        placemark=0
+        newstring=""
+        line=line.rstrip()
+        #getting rid of \'s
+        if 'B' in line:
+            continue
+        if '@' in line:
+            continue
+        if '}' in line:
+            continue
+        if '+' in line:
+            continue
+        if '-' in line:
+            continue
+        if ';' in line:
+            continue
+        #really sloppily discounting any lines apart from those with basepairs
+        totalbase=len(line)
+        totaltotalbase+=totalbase
+        Pscoreplacemark=1
+        print Pscorelist
+        for basepair in line:
+            placemark+=1
+            errorchance=Pscorelist[Pscoreplacemark-1]
+            int(errorchance)
+            genedice=random.random()
+            #checks place in line then produces a random number based on it
+            if genedice > errorchance:
+                newstring+=basepair
+                continue
+                #if the number generated is higher than the produced error chance, the base pair is added to a new line
+            else:
+                errorcount+=1
+                if basepair==('G'):
+                    if genedice < errorchance/3:
+                        newstring+='A'
+                        continue
+                    if genedice < 2*errorchance/3:
+                       newstring+='C'
+                       continue
+                    if genedice <= errorchance:
+                        newstring+='T'
+                        continue
+                if basepair==('C'):
+                    if genedice < errorchance/3:
+                        newstring+='G'
+                        continue
+                    if genedice < 2*errorchance/3:
+                       newstring+='A'
+                       continue
+                    if genedice <= errorchance:
+                        newstring+='T'
+                        continue
+                if basepair==('A'):
+                    if genedice < errorchance/3:
+                        newstring+='G'
+                        continue
+                    if genedice < 2*errorchance/3:
+                        newstring+='C'
+                        continue
+                    if genedice <= errorchance:
+                        newstring+='T'
+                        continue
+                if basepair==('T'):
+                    if genedice < errorchance/3:
+                        newstring+='A'
+                        continue
+                    if genedice < 2*errorchance/3:
+                        newstring+='C'
+                        continue
+                    if genedice <= errorchance:
+                        newstring+='G'
+                        continue
+                    #if the generated number is lower than the error chance, then a random base pair is put in its place on the list
+        print newstring
+        print errorcount, 'errors'
```

Comment:
Messages like these should be printed to stderr instead of stdout. http://www.diveintopython.net/scripts_and_streams/stdin_stdout_stderr.html

This program should only print out a valid fastq line so it can be used downstream.